marlowe/test/Spec/Marlowe/Marlowe.hs

```diff
@@ -67,7 +67,7 @@ getSignaturesTest :: IO ()
 getSignaturesTest = do
     let alicePk = walletPubKey alice
     let bobPk = walletPubKey bob
-    let deposit pk = IDeposit (AccountId 0 alicePk) pk (Ada.adaOf 256)
+    let deposit pk = IDeposit (AccountId 0 alicePk) pk "" "" 256_000000
```

Comment:
why are these all `""`?

Reply:
Those are adaSymbol and adaToken :) There are `IsString` instances for those types.

Reply:
Why not just use `adaSymbol` and `adaToken`? We have defined values for those precisely so you don't need to write `""` everywhere.